ansible/playbooks/tls/issue-certs.yml

```diff
@@ -0,0 +1,17 @@
+- hosts: redpanda
```

Comment:
It's split up into generate-csr, issue-certs, and install-certs because the middle step (issue-certs) could be done externally, using a corporate CA or whatever...